hoodie-client/src/test/java/com/uber/hoodie/common/HoodieClientTestUtils.java

```diff
@@ -113,6 +113,7 @@ public static void fakeDataFile(String basePath, String partitionPath, String co
   public static SparkConf getSparkConfForTest(String appName) {
     SparkConf sparkConf = new SparkConf().setAppName(appName)
         .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+        .set("spark.driver.allowMultipleContexts","true")
```

Comment:
We should be able to run tests without this, multipleContexts is a buggy feature, let's try to avoid using this..

Reply:
> We should be able to run tests without this, multipleContexts is a buggy feature, let's try to avoid using this..

ok, thanks